flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *
+from future.moves.urllib.parse import urlparse
+
+import logging
+from collections import MutableSet
+
+import requests
+from requests import RequestException
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+
+log = logging.getLogger('ombi_list')
+
+def generate_title(item, strip):
+    if item.get('releaseDate') and not strip:
```

Comment:
releaseDate for an episode is the air date I take it? Won't make a whole lot of sense to include the air date in the title.

Reply:
When taken from Ombi, it refers ether to the Movie release date or a Series Release date.

It is not used for an episode title nor does it refer to an episode airdate (maybe the first airdate of a series), at least in regards to ombi.

Most of the time thetvdb or imbdb it will be used for matching but when using the title for search of a TV series it is important to differentiate Doctor Who (2005) and Doctor Who (1963) when the list is used with discover and only the title is used.